fabric/patches/2_0004-Deploy-qscc-via-opt_scc-extension-point.patch

```diff
@@ -0,0 +1,86 @@
+From 0505d4c4c4f7ac665afced25975840be299fe9dc Mon Sep 17 00:00:00 2001
+From: michael steiner <michael.steiner@intel.com>
+Date: Wed, 19 Aug 2020 15:40:39 -0700
+Subject: [PATCH 1/4] Deploy qscc via opt_scc extension point
```

Comment:
any reason this shows as [Patch 1/4] even though it seems to be the fourth patch?

Reply:
good catch: i have to manually edit that given the way i create these format patches. Should (will) be fixed now ...